LICENSE-APACHE

```diff
@@ -220,3 +220,15 @@ Apache License
 
 
    END OF TERMS AND CONDITIONS
+
+---
+
+The contents of snippets.json are derived from Ruby LSP:
```

Comment:
:wave: @vinistock, are you ok with this attribution?

Reply:
Let's include the copyright line.

```
Copyright (c) 2022-present, Shopify Inc.
https://github.com/Shopify/ruby-lsp
  
  Released under the MIT license
  https://github.com/Shopify/ruby-lsp/blob/main/vscode/LICENSE.txt
```